bundler/doc/contributing/README.md

```diff
@@ -18,11 +18,11 @@ To request substantial changes to Bundler and/or Bundler documentation, please r
 
 Here are the different ways you can start contributing to the Bundler project:
 
-* [First contribution suggestions](https://github.com/rubygems/bundler/blob/master/doc/contributing/HOW_YOU_CAN_HELP.md)
-* [Adding new features](https://github.com/rubygems/bundler/blob/master/doc/development/NEW_FEATURES.md)
-* [Triaging bugs](https://github.com/rubygems/bundler/blob/master/doc/contributing/BUG_TRIAGE.md)
-* [Writing documentation](https://github.com/rubygems/bundler/blob/master/doc/documentation/WRITING.md)
-* [Community engagement](https://github.com/rubygems/bundler/blob/master/doc/contributing/COMMUNITY.md)
+* [First contribution suggestions](/contributing/HOW_YOU_CAN_HELP.md)
+* [Adding new features](/development/NEW_FEATURES.md)
+* [Triaging bugs](/contributing/BUG_TRIAGE.md)
+* [Writing documentation](/documentation/WRITING.md)
+* [Community engagement](/contributing/COMMUNITY.md)
```

Comment:
Not sure how relative links work here, the these seem to lead to 404's for me? I think

```suggestion
* [Community engagement](/bundler/doc/contributing/COMMUNITY.md)
```

instead would work?

(Same for the others)

Reply:
Will change them.
The relative links worked for me, I'm guessing my IDE figured where to link them.

Reply:
What I did was go to the PR diff, and clicked on the "three dots" menu on the top right of the changed file, and then on "View file". For there I could see the file rendered with the changes in this PR. Clicking on the links goes to a missing page from there.